monai/apps/pathology/transforms/post/array.py

```diff
@@ -379,6 +379,7 @@ def _generate_contour_coord(self, current: np.ndarray, previous: np.ndarray) ->
         """
 
         p_delta = (current[0] - previous[0], current[1] - previous[1])
+        row, col = -1, -1
```

Comment:
And also here?

Reply:
The code checkers don't like variables that only exist in some code branches.